src/packages/migrate/src/MigrateEngine.ts

```diff
@@ -221,40 +223,51 @@ export class MigrateEngine {
 
         this.child.on('error', (err) => {
           console.error('[migration-engine] error: %s', err)
-          reject(err)
           this.rejectAll(err)
+          reject(err)
```

Comment:
Unrelated, but it looks wrong. Wdyt?

Reply:
It's easier to read with the `reject` last but since it's asynchronous it should not matter during runtime.